src/components/header.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const propTypes = {
```

Comment:
These propTypes are a way to explicitly define what _sort_ of properties your component is expecting. Again if we think of a component as a box with holes in it that specific things are let into - ie, the hole might be the property 'names' and what it _expects_ to be passed into this hole is an array of names. 

Now if we for example passed in a number, all the code in our component that expects an array will suddenly freak out, because it can't run .map on a number or .length on a number or whatever.

So what propTypes are doing are saying, hey - for the property 'title' I specifically expect to get a string.

This might be an interesting topic, and maybe I'll include the babel library that lets you more declaratively set expected property types for this project - it's pretty painless and it makes your classes prettier